store/localstore/snapshot.go

```diff
@@ -32,9 +30,27 @@ var (
 
 type dbSnapshot struct {
 	db      engine.DB
+	rawIt   engine.Iterator
 	version kv.Version // transaction begin version
 }
 
+func (s *dbSnapshot) internalSeek(startKey []byte) (engine.Iterator, error) {
+	if s.rawIt == nil {
+		var err error
+		s.rawIt, err = s.db.Seek([]byte{0})
+		if err != nil {
+			return nil, errors.Trace(err)
+		}
+	}
+	ok := s.rawIt.Seek(startKey)
+	if !ok {
+		s.rawIt.Release()
```

Comment:
s.Release is dbSnapshot's method, in internalSeek, I just want to release rawIt.